<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_5">
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>131</width>
       <height>211</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>157</height>
       </rect>
      </property>
      <attribute name="label">
       <string>人</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>157</height>
       </rect>
      </property>
      <attribute name="label">
       <string>shi</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>-190</x>
       <y>-81</y>
       <width>391</width>
       <height>341</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>389</width>
        <height>339</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="b1">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新建项目</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
